tests/settings.py

```diff
@@ -62,7 +62,8 @@
 USE_TZ = True
 USE_I18N = False
 USE_L10N = False
-STATIC_URL = '/static/'
+
+STATIC_URL = os.path.abspath('django_freeradius/static') + '/'
```

Comment:
Are these changes necessary?

Reply:
Yeah, they are necessary to achieve a correct path styles and js for the pdf

Reply:
So you're saying it doesn't work at all with `STATIC_URL = '/static/'`